### ISSUE TITLE
the `extensionality` tactic (added in 8.7) is undocumented

### ISSUE THREAD

**Issue by gasche** [coq/coq#8839]
#### Version

I'm looking at the [reman 8.8.2 tactic index](https://coq.inria.fr/distrib/V8.8.2/refman/coq-tacindex.html).

#### Description of the problem

The `extensionality` tactic, added in 8.7 (according to CHANGES) is undocumented.

**Comment by Zimmi48** [comment]
cc @herbelin

**Comment by gasche** [comment]
I realize now that the tactic is only defined in the FunctionalExtensionality module, so maybe it is intentional that those kind of module-specific tactics are not found in the tactic index?

Also I'm not sure anymore when `extensionality` was introduced; 8.7 introduced `extensionality in <hyp>`, but I was looking for documentation of the form `extensionality <ident>` which I think comes from @mattam82 in 2008: 640a6d2f48ba07b51bcabc4235eaa4a497f4c263.

**Comment by Zimmi48** [comment]
>I realize now that the tactic is only defined in the FunctionalExtensionality module, so maybe it is intentional that those kind of module-specific tactics are not found in the tactic index?

I don't think so. Many tactics require importing a module first and yet they are documented in the manual.

**Comment by SkySkimmer** [comment]
Should this issue be moved to the stdlib repo?
